tests/blob.rs

```diff
@@ -22,10 +29,11 @@ fn blob_too_small_test() {
 #[test]
 fn blob_test_fail() {
     //  Invalid PNG header (0x51 instead of 0x50)
-    let data = vec![0x89, 0x51, 0x4E, 0x47, 0x0D, 0x0A, 0x1A, 0x0A, 
-                    0x00, 0x00, 0x00, 0x0D, 0x49, 0x48, 0x44, 0x52, 
-                    0x00, 0x00, 0x00, 0x7B, 0x00, 0x00, 0x01, 0x41,
-                    0x08, 0x06, 0x00, 0x00, 0x00, 0x9A, 0x38, 0xC4];
+    let data = vec![
+        0x89, 0x51, 0x4E, 0x47, 0x0D, 0x0A, 0x1A, 0x0A, 0x00, 0x00, 0x00, 0x0D, 0x49, 0x48, 0x44,
```

Comment:
Not a fan of how many it's deciding to put on a single line here because it makes it harder to track things in 4 byte increments. Previous was 8 per row, but this is 15.